src/Compilers/VisualBasic/Portable/Symbols/Source/SourceMemberContainerTypeSymbol.vb

```diff
@@ -1802,9 +1802,8 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
                             Dim candidateLocation As Location = candidate.NonMergedLocation
                             Debug.Assert(candidateLocation IsNot Nothing)
 
-                            If partialMethods.Contains(candidate) Then
-                                '  partial-partial conflict
-                                partialMethods.Remove(candidate)
+                            '  partial-partial conflict
```

Comment:
>'  partial-partial conflict [](http://example.com/codeflow?start=28&length=27)

The comment should probably stay inside the `If`

Reply:
Moved back in [28f42ef](https://github.com/dotnet/roslyn/pull/69179/commits/28f42ef0063ee7a30ac421a352a9c1e11a101f52).